src/xo-app/vm/tab-advanced.js

```diff
@@ -252,6 +253,28 @@ export default ({
               <AffinityHost vm={vm} />
             </td>
           </tr>
+          {vm.virtualizationMode === 'hvm' &&
+            <tr>
+              <th>{_('vmVGA')}</th>
+              <td>
+                <Toggle value={vm.vga === 'std'} onChange={value => editVm(vm, { vga: value ? 'std' : 'cirrus' })} />
+              </td>
+            </tr>
+          }
+          {vm.vga === 'std' &&
+            <tr>
+              <th>{_('vmVideoram')}</th>
+              <td>
+                <select
+                  className='form-control'
+                  onChange={event => editVm(vm, { videoram: +event.target.value })}
```

Comment:
Use `getEventValue()`.

And I don't think it's necessary to cast `videoram` to a number.